# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: caps/BasePrincipal.h
# Commit: 532731e94bb2
# Full Hash: 532731e94bb23b01b5e7136bee829844a0f08bc2
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2020-06-01 21:42:28
# Regressor Bug: 1638358
# File Overlap Count: 1
# Description:
#   Bug 1638358 - Cookie Schemeful Same-Site - part 5 - schemeful comparison, r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D75629
# ==============================================================================

diff -r fad2ba760157 -r 532731e94bb2 caps/BasePrincipal.h
--- a/caps/BasePrincipal.h	Mon Jun 01 11:43:30 2020 +0000
+++ b/caps/BasePrincipal.h	Mon Jun 01 11:43:28 2020 +0000
@@ -122,6 +122,7 @@
   NS_IMETHOD GetIsContentPrincipal(bool* aResult) override;
   NS_IMETHOD GetIsExpandedPrincipal(bool* aResult) override;
   NS_IMETHOD GetIsSystemPrincipal(bool* aResult) override;
+  NS_IMETHOD GetScheme(nsACString& aScheme) override;
   NS_IMETHOD SchemeIs(const char* aScheme, bool* aResult) override;
   NS_IMETHOD IsURIInPrefList(const char* aPref, bool* aResult) override;
   NS_IMETHOD IsL10nAllowed(nsIURI* aURI, bool* aResult) override;